crawl-ref/source/dat/strings/de/monsters-plural-counted-nom.txt

```diff
@@ -1660,7 +1660,7 @@
 %d Oklob-Setzling-Zauberer
 %%%%
 %d oklob plant conjurers
-%d Oklob-Pflanze-Hervorrufer
+%d Oklob-Pflanze-Konjurant
```

Comment:
Oklobpflanzenkonjurant

Reply:
There's a lot of this. I'll handle it in a separate PR.